Extension/src/Debugger/configurationProvider.ts

```diff
@@ -679,6 +679,7 @@ export function buildAndDebugActiveFileStr(): string {
 
 export async function buildAndDebug(textEditor: vscode.TextEditor, cppVsDbgProvider: CppVsDbgConfigurationProvider | null, cppDbgProvider: CppDbgConfigurationProvider, debugModeOn: boolean = true): Promise<void> {
     const folder: vscode.WorkspaceFolder | undefined = vscode.workspace.getWorkspaceFolder(textEditor.document.uri);
+    const event: string = debugModeOn ? "buildAndDebug" : "buildAndRun";
```

Comment:
Rather than have two different event names, I find it easier to query for Properties when I do the reporting.  I think it would be better if we have one event name, maybe something that matches what we're calling this (e.g. "launchShortcut"), and then add a "type" property where you can put "buildAndDebug" or "buildAndRun" as the values.